Makefile

```diff
@@ -1,31 +1,28 @@
-.PHONY: asset cache composer coverage cs database infection integration it test test-env unit
+.PHONY: asset cache coverage cs database infection integration it test test-env unit
 
 it: cs test
 
 asset:
 	yarn install
 	yarn run production
 
-cache:
+cache: vendor
```

Comment:
This target should probably depend on `vendor` as well as it would fail in the worst case (no dependencies installed).